types/lodash/common/lang.d.ts

```diff
@@ -1,5 +1,5 @@
 import _ = require("../index");
-// eslint-disable-next-line @definitelytyped/strict-export-declare-modifiers
+// eslint-disable-next-line @definitelytyped/strict-export-declare-modifiers, @typescript-eslint/no-unsafe-function-type
 type GlobalFunction = Function;
```

Comment:
I got a lint error here, but I believe it's safe to ignore it.

![image](https://github.com/user-attachments/assets/366c66a9-e78b-42c0-9c26-dc374069c28c)